src/nvim/eval.c

```diff
@@ -15827,11 +15827,11 @@ static void f_sign_getplaced(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 {
   buf_T *buf = NULL;
   dict_T *dict;
-  dictitem_T *di;
```

Comment:
`di` and `lnum` are the standard terms used in the codebase, for better and worse. I do not think we want to change them in isolation in a single function. (also for sign code in particular, we have yet some vim patches to merge, which we want to do first before doing refactors here).

Reply:
@bfredl Issue fixed as suggested